cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/ghcr-util.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+
+# Copyright 2021-2022 the Kubeapps contributors.
```

Comment:
Nitpick: technically, since the file has been created now, this is the proper header:

```suggestion
# Copyright 2022 the Kubeapps contributors.
```